src/__tests__/translit.test.js

```diff
@@ -13,6 +14,22 @@ describe('translit', () => {
     expect(JSON.stringify(translit('lwl rMgu iqs kau lgw ijs ky vfBwgw ] mYlw kdy n hoveI nh lwgY dwgw ]1]')))
       .toBe('"laal ra(n)g tis kau lagaa jis ke vaddabhaagaa || mailaa kadhe na hoviee neh laagai dhaagaa ||1||"');
   });
+  it('Should return only english transliteration of Gurmukhi', () => {
+    expect(JSON.stringify(translit('AMqir iprI ipAwru ikau ipr ibnu jIvIAY rwm ]')))
+      .toBe('"a(n)tar piree piaar kiau pir bin jeeveeaai raam ||"');
+  });
+  it('Should return only english transliteration of Gurmukhi', () => {
+    expect(JSON.stringify(translit('jb lgu drsu n hoie ikau AMimRqu pIvIAY rwm ]')))
+      .toBe('"jab lag dharas na hoi kiau a(n)mrit peeveeaai raam ||"');
+  });
+  it('Should return only english transliteration of Gurmukhi', () => {
+    expect(JSON.stringify(translit('hir hir nwmu iDAweIAY ijs nau ikrpw kry rjwie ]3]')))
+      .toBe('"har har naam dhiaaieeaai jis nau kirapaa kare rajai ||3||"');
+  });
+  it('Should return only english transliteration of Gurmukhi', () => {
```

Comment:
nit picking, but every `it` test case should have a more meaningful and unique name. each one of these have `Should return only english transliteration of Gurmukhi` and doesn't really convey what is being tested.

Reply:
updated